src/graph/helpers/create-properties.ts

```diff
@@ -0,0 +1,55 @@
+import { generate } from '~/src/id.js';
+import { Relation } from '~/src/relation.js';
+import { Triple } from '~/src/triple.js';
+import type { PropertiesParam } from '~/src/types.js';
+import type { Op } from '../../types.js';
+
+type CreatePropertiesParams = {
+  entityId: string;
+  properties: PropertiesParam;
+};
+
+export const createProperties = (params: CreatePropertiesParams) => {
+  const { entityId, properties } = params;
+  const ops: Op[] = [];
+
+  for (const [attributeId, property] of Object.entries(properties)) {
+    if ('type' in property) {
+      const propertyTripleOp = Triple.make({
+        entityId,
+        attributeId,
+        value: property,
+      });
+      ops.push(propertyTripleOp);
+    } else if ('to' in property) {
+      const relationId = property.id ?? generate();
+      const propertyRelationOp = Relation.make({
+        relationId,
```

Comment:
The `relationId` argument is the entity id of the Relation entity, not the Property entity. It looks like you're setting the id of the property to be the same as the new relation entity. Is that on purpose?

Reply:
Byron (id 1) - Works at -> Geo (id 2). The `Works at` entity might have an id of 3, but the Relation entity representing "Byron works at Geo" might have an id of 4.

Check out this entity an example. The entity id for this relation is `VbYuNDM2S9LKyij73Aj5CA`. The entity id for the `Works at` property is `U1uCAzXsRSTP4vFwo1JwJG`.
https://www.geobrowser.io/space/6tfhqywXtteatMeGUtd5EB/VbYuNDM2S9LKyij73Aj5CA

Reply:
That's exactly what I intended to do. For a property you can pass in the `to` ID and if you want to have control of the entity ID of the relation you can pass in `id`.

Without `id`:
> Byron (id 1) - Works at (generated internally) -> Geo (id 2).

With `id` set to `6`:
> Byron (id 1) - Works at (id 6) -> Geo (id 2).

Reply:
So in this case `property.id` is meant to be the id of the _relation_ and not the property?